lib/muffin/frostings/attributes.rb

```diff
@@ -63,5 +63,9 @@ class NestedAttribute
     def initialize(attributes)
       self.attributes = attributes
     end
+
+    def persisted?
+      attributes[:id].present?
+    end
```

Comment:
So you wanted to say to better move this to the `Attributes` modul because it's included in both classes,. Can do this :+1: